File: NonFactors/NonFact.js
/*
    Author : Pakshal Jain
    Date : 04/10/2022
    Program : To Print Nonfactors of Entered Number
*/
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//Function To Find Nonfactors of Entered Number
function NonFact(no) {
    var i = 0;
    for (i = 1; i < no; i++) {
        if (no % i != 0) {
            console.log("%d", i);
        }
    }
}
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//main function
function main() {
    var No = 10;
    console.log("Ganapti Bappa Morya...");
    //Call To Nonfact Function
    NonFact(No);
}
//Call To main Function
main();

File: NonFactors/NonFact.java
/*
    Author : Pakshal Jain
    Date : 04/10/2022
    Program : To Print Nonfactors of Entered Number
*/

//Required Libraries
import java.lang.*;
import java.util.*;

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Class Declaration
class Find
{
    //Characteristics of Class
    int No = 0;

    //Constructor of Class
    Find(int no)
    {
        this.No = no;
    }
    //Behaviour of Class
    void nonfact()
    {
        int i = 0;

        System.out.println("Nonfactors of Entered Number is As Follows");

        for(i = 1;i < this.No;i++)
        {
            if(this.No % i != 0)
            {
                System.out.print(i+"\t");
            }
        }
    }
}

///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

class NonFact
{
    public static void main(String args[])
    {
        int No = 0;
        
        Scanner sobj = new Scanner(System.in);
        System.out.println("Jay Ganesh....");
        
        System.out.println("Ente One Number");
        No = sobj.nextInt();
        //Creating object of Class
        Find fobj = new Find(No);
        //Calling Behaviour of Class
        fobj.nonfact();       
    }
}